Formula/lib/libgrapheme.rb

```diff
@@ -4,7 +4,7 @@ class Libgrapheme < Formula
   url "https://dl.suckless.org/libgrapheme/libgrapheme-2.0.2.tar.gz"
   sha256 "a68bbddde76bd55ba5d64116ce5e42a13df045c81c0852de9ab60896aa143125"
   license "ISC"
-  head "git://git.suckless.org/libgrapheme/", branch: "master"
+  head "https://git.suckless.org/libgrapheme/", using: :git, branch: "master"
```

Comment:
Why?

Reply:
Because the git protocol is cleartext, unencrypted and unauthenticated. Like HTTP.